### PR TITLE
Translated untranslated phrases

### PR DESCRIPTION
**Pull request by matsduf**
Two phrases that were found untranslated.

It is, however, unclear if and when those are outputted. Especially, "The domain name contains unauthorized character" cannot be found in any code.

### CODE REVIEW

File: src/assets/i18n/sv.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
...
   ,"About domain name system, DNS": "Om domain name system, DNS"
   ,"About DNS Text":"De flesta som använder internet har aldrig ens reflekterat över vad som egentligen händer när man har skrivit in en adress i sin webbläsare och tryckt ned enter-tangenten. Något förenklat är domännamnssystemet (DNS) en databas som översätter domännamn till IP-adresser, på samma sätt som en telefonkatalog översätter namn till telefonnummer. En IP-adress är en unik nummerserie som identifierar varje dator som är exponerad mot internet. Det kan liknas vid hur varje telefon har ett eget telefonnummer i telenätet. Långa nummerserier fungerar fint för datorer, men för oss människor är de svåra att memorera. Därför har man utvecklat DNS ett extra system för oss när vi vill nå fram till en webbserver med en viss sajt på eller en e-postserver som vi vill skicka e-postmeddelanden till."
-  ,"No data for this test": "No data for this test"
-  ,"The domain name contains unauthorized character(s)": "The domain name contains unauthorized character(s)"
+  ,"No data for this test": "Ingen data för detta test"
+  ,"The domain name contains unauthorized character(s)": "Domännamnet innehåller otillåtet tecken"
```
Comment (mattias-p) [inline src/assets/i18n/sv.json:29]: Weird grammar. I'd prefer "minst ett otillåtet".

Comment (matsduf) [reply src/assets/i18n/sv.json:29]: But the string is probably never used anyway.

**Comment by matsduf** [comment]
@pamasse, can you take a look at this?

File: src/assets/i18n/sv.json (hunk 1 above)
Comment (mattias-p) [reply src/assets/i18n/sv.json:29]: Should we keep it if it's not used?

Comment (matsduf) [reply src/assets/i18n/sv.json:29]: No, and that this what I've asked @pamasse to check. If not used, it should be removed from all the json files.

**Comment by sandoche2k** [comment]
@pamasse merge?

**Pull request merged**